examples/math/config.go

```diff
@@ -0,0 +1,47 @@
+package math
```

Comment:
Yup, I've seen multiconfig, just thought external dependency may be too much for an example.